### PR TITLE
Adding bulk acceptance and rejection of detailed reports.

### PR DESCRIPTION
**Pull request by jerdzica**

### CODE REVIEW

File: addons/bestja_detailed_reports/views/wizards.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                        </group>
+                    </div>
+                    <div attrs="{'invisible': [('user_can_moderate', '=', True)]}">
+                        <h3>Nie masz uprawnień do akceptacji raportów!</h3>
+                    </div>
+                    <footer>
+                        <div attrs="{'invisible': [('user_can_moderate', '=', False)]}">
+                            <button name="accept_detailed_reports" type="object" string="Tak, chcę zaakceptować" class="oe_highlight"/>
+                            lub
+                            <button special="cancel" string="Nie" class="oe_link"/>
```
Comment (ludwiktrammer) [inline addons/bestja_detailed_reports/views/wizards.xml]: Wszędzie mamy "Anuluj", dlaczego tu jest "Nie"?

**Comment by ludwiktrammer** [comment]
:+1: :sunny:

File: addons/bestja_detailed_reports/wizards.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        self.reports.set_accepted()
+
+    @api.one
+    def reject_detailed_reports(self):
+        self.reports.set_rejected()
+
+    @api.one
+    @api.depends('reports')
+    def _compute_all_user_can_moderate(self):
+        self.user_can_moderate = all([report.user_can_moderate for report in self.reports])
```
Comment (KrzysiekJ) [inline addons/bestja_detailed_reports/wizards.py]: Brackets are unnecessary. I’ll fix it when merging.

**Comment by KrzysiekJ** [comment]
Merged in ad63098.

**Pull request closed without merging**